src/eventDispatcher.js

```diff
@@ -0,0 +1,19 @@
+function EventDispatcher () {
+	this.eventDispatcherMap = {};
+};
+
+EventDispatcher.prototype.bind = function(nameOfFunction, whichFunction) {
+	if(this.eventDispatcherMap[nameOfFunction]) {
+		this.eventDispatcherMap[nameOfFunction].push(whichFunction);
+	} else {
+		this.eventDispatcherMap[nameOfFunction] = [whichFunction];
+	}
+};
+
+EventDispatcher.prototype.trigger = function(nameOfFunction) {
+	for (var i = 0; i < this.eventDispatcherMap[nameOfFunction].length; i++) {
```

Comment:
What will happen in the following scenario:
```
var eventDispatcher = new EventDispatcher();
eventDispatcher.trigger("exampleEvent");
```
?

Reply:
In that case you have a error message in console : 'Uncaught TypeError: Cannot read property 'length' of undefined' because you didn't make function bind with this eventName.